src/test/resources/org/jenkinsci/plugins/simpletheme/ConfigurationAsCode.yml

```diff
@@ -1,6 +1,6 @@
 ---
-unclassified:
-  simple-theme-plugin:
+appearance:
```

Comment:
Since this changes, would it make sense to mark this version as incompatible with previous versions?